src/mumble/AudioOutputSample.cpp

```diff
@@ -214,20 +218,18 @@ bool AudioOutputSample::prepareSampleBuffer(unsigned int snum) {
 		return true;
 
 	// Calculate the required buffersize to hold the results
-	unsigned int iInputFrames = static_cast<unsigned int>(ceilf(static_cast<float>(snum * sfHandle->samplerate()) / static_cast<float>(iOutSampleRate)));
+	unsigned int iInputFrames = static_cast<unsigned int>(ceilf(static_cast<float>(frameNumber * sfHandle->samplerate()) / static_cast<float>(iOutSampleRate)));
 	unsigned int iInputSamples = iInputFrames * sfHandle->channels();
 
-	bool mix = sfHandle->channels() > 1;
 	STACKVAR(float, fOut, iInputSamples);
-	STACKVAR(float, fMix, iInputFrames);
 
 	bool eof = false;
 	sf_count_t read;
 	do {
 		resizeBuffer(iBufferFilled + snum);
 
-		// If we need to resample or mix write to the buffer on stack
-		float *pOut = (srs || mix) ? fOut : pfBuffer + iBufferFilled;
+		// If we need to resample, write to the buffer on stack
+		float *pOut = (srs) ? fOut : pfBuffer + iBufferFilled;
```

Comment:
Why is the need for mixing gone at this point?

Reply:
Cuz now we can play stereo samples directly, so we don't have to downmix them into mono stream xD

Reply:
Ah so that's what this was for. Thanks! :)